Nos/Views/New Note/ComposerActionBar.swift

```diff
@@ -53,16 +55,40 @@ struct ComposerActionBar: View {
                             endUploadingImage()
                             print("error uploading: \(error)")
                             
-                            alert = AlertState(title: {
-                                TextState(String(localized: .imagePicker.errorUploadingFile))
-                            }, message: {
-                                if case let FileStorageAPIClientError.uploadFailed(message) = error,
-                                    let message {
-                                    TextState(String(localized: .imagePicker.errorUploadingFileWithMessage(message)))
-                                } else {
-                                    TextState(String(localized: .imagePicker.errorUploadingFileMessage))
-                                }
-                            })
+                            alert = AlertState<ComposerActionBar.AlertAction>(
+                                title: {
+                                    if case FileStorageAPIClientError.fileTooBig = error {
+                                        return TextState(
+                                            String(localized: .imagePicker.errorUploadingFileExceedsSizeLimit)
+                                        )
+                                    } else {
+                                        return TextState(String(localized: .imagePicker.errorUploadingFile))
+                                    }
+                                }(),
+                                message: {
+                                    if case let FileStorageAPIClientError.fileTooBig(message) = error, let message = message {
```

Comment:
Swift has shorthand for this:
```suggestion
                                    if case let FileStorageAPIClientError.fileTooBig(message) = error, let message {
```